src/query.ts

```diff
@@ -29,6 +29,13 @@ export type ParsedQuery = Record<string, string | string[]>;
  *
  * The input can be a query string with or without the leading `?`.
  *
+ * @example
+ *
+ * ```js
+ * parseQuery("?foo=bar&baz=qux");
+ * // { foo: "bar", baz: "qux" }
+ * ```
+ *
```

Comment:
It would be helpful to describe how array is handled, as you already did for `encodeQueryItem()`.